packages/portal-proto/src/features/repositoryApp/appApi.ts

```diff
@@ -18,11 +19,25 @@ const downloadAppReducers = combineReducers({
   facetRanges: repositoryRangeFacetsReducer,
 });
 
+const migrations = {
+  2: (state) => {
+    return {
+      ...state,
+      facets: {
+        customFacets: state.facets.facets.filter(
+          (facet) => !RepositoryDefaultConfig.facets.includes(facet),
+        ),
```

Comment:
This allows us to alter the store to only store the custom facets for users who already have a persisted store.